src/screens/HomePage/HomePage.style.tsx

```diff
@@ -18,16 +18,42 @@ export const Content = styled.div`
   margin: auto auto 3rem;
 `;
 
-export const Section = styled.div`
+export const HomePageBlock = styled.div`
   margin: 3.5rem 1rem;
+
+  @media (min-width: ${materialSMBreakpoint}) {
+    margin: 3.5rem 1rem 5rem;
+  }
+`;
+
+export const HomePageBlockHeader = styled.h2`
+  ${props => props.theme.fonts.regularBookBold};
+  font-size: 26px;
+  margin-top: 15px;
+  text-align: center;
+
+  @media (min-width: ${materialSMBreakpoint}) {
+    margin-top: 5px;
+    font-size: 36px;
+  }
+`;
+
+export const HomePageBlockSubtitle = styled.div`
+  ${props => props.theme.fonts.regularBook};
+  font-size: 16px;
+  margin-top: 20px;
+  text-align: center;
+
+  @media (min-width: ${materialSMBreakpoint}) {
+    margin-top: 24px;
+  }
 `;
 
 // zero right margin so that it's full bleed on mobile when overflowing
 export const RegionItemsWrapper = styled.div`
   display: flex;
   flex-direction: row;
   margin-top: 0.75rem;
-  margin-bottom: 2.25rem;
```

Comment:
I think this was left over from when geolocation cards could wrap on mobile and we needed vertical space between them.  Removing it made it easier to get the rest of the margins consistent.